src/Explorer/Menus/CommandBar/CommandBarComponentButtonFactory.tsx

```diff
@@ -275,6 +285,22 @@ function createOpenSynapseLinkDialogButton(container: Explorer): CommandButtonCo
   };
 }
 
+function createLoginForEntraIDButton(container: Explorer): CommandButtonComponentProps {
```

Comment:
We should only display the login button when we know that we are using RBAC and we don't have the token. i.e. the button should disappear after user has logged in.